dbt_subprojects/dex/models/_projects/zeroex/ink/zeroex_ink_settler_addresses.sql

```diff
@@ -0,0 +1,10 @@
+{{
+    config(
+        schema = 'zeroex',
+        alias = 'ink_settler_addresses',
+        materialized = 'table'
```

Comment:
we should be changing this config block to be incremental. you can set this up similar to v2 trades config block below and just update the unique keys and such. 

plz do apply to all other chains for settler address models in this PR too, so we can get it all done in one